internal/compress/zstd_option.go

```diff
@@ -46,7 +46,7 @@ func WithZstdGob(opts ...GobOption) ZstdOption {
 // WithZstdCompressionLevel represents the option to set the compress level for zstd.
 func WithZstdCompressionLevel(level int) ZstdOption {
 	return func(c *zstdCompressor) error {
-		c.eoptions = append(c.eoptions, zstd.WithEncoderLevel(zstd.EncoderLevelFromZstd(level)))
+		c.eoptions = append(c.eoptions, zstd.WithEncoderLevel(level))
```

Comment:
When will it execute `zstd.EncoderLevelFromZstd()` function which convert the level to the internal used level?

Reply:
@kevindiu 
sorry. I forgot to push options.go:bow:
The conversion is done in the line below
https://github.com/vdaas/vald/blob/test/internal/compress/gzip/internal/compress/zstd/option.go#L6-L10